internal_tests/use_cases/all_use_cases.txt

```diff
@@ -129,6 +129,7 @@ Category: s2s
 9:: UserScript_obsERA_obsOnly_OMI:: model_applications/s2s/UserScript_obsERA_obsOnly_OMI.conf:: spacetime_env, metdatadb
 10:: UserScript_obsERA_obsOnly_RMM:: model_applications/s2s/UserScript_obsERA_obsOnly_RMM.conf:: spacetime_env, metdatadb
 11:: UserScript_fcstGFS_obsERA_WeatherRegime:: model_applications/s2s/UserScript_fcstGFS_obsERA_WeatherRegime.conf:: weatherregime_env,cartopy,metplus
+11:: UserScript_obsERA_obsOnly_Stratosphere:: model_applications/s2s/UserScript_fcstGFS_obsERA_Stratosphere.conf:: metplotpy_env,metplus
```

Comment:
There are 2 entries under s2s that have index 11. Change the 2nd one to 12. Also, the use case will not run in the automated test suite unless it is added as a new group to .github/parm/use_case_groups.json:

> 
>   {
>     "category": "s2s",
>     "index_list": "12",
>     "run": true
>   },

Setting run to true will run this use case group for each push to the branch. This can be used for testing the new use case. When the use case is running without issues, change run to false so that it does not run for other feature branches. More info here: https://metplus.readthedocs.io/en/develop/Contributors_Guide/add_use_case.html#add-new-category-to-test-runs